CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ This page provides information about contributing code to the Jenkins core codeb
    - Apache Maven 3.8.1 or above. You can [download Maven here](https://maven.apache.org/download.cgi).
      In the Jenkins project we usually use the most recent Maven release.
    - Any IDE which supports importing Maven projects.
-   - Install [NodeJS 16.x](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
+   - Install [Node.js 18.x](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
      - Frontend tasks are run using [yarn](https://yarnpkg.com/). Run `npm install -g yarn` to install it.
```

Comment:
Accoring to the [docs](https://yarnpkg.com/getting-started/install#install-corepack) it should be installed with corepack:

```suggestion
     - Frontend tasks are run using [yarn](https://yarnpkg.com/). Run `corepack enable` to install corepack as the recommended way and afterwards `corepack prepare yarn@stable --activate` to install yarn.
```

Reply:
`corepack enable` from the war directory should be all that's needed as the version is defined in the package.json https://github.com/jenkinsci/jenkins/blob/f0952735b523ff5af6aab3f0a7b214964b5a16a9/war/package.json#L68

Reply:
Oh, that is nice. I was not aware of that -  tested it and it works fine